### PR TITLE
Some more changes of nodes to outputs

### PR DESCRIPTION
**Pull request by diyessi**

### CODE REVIEW

File: src/ngraph/op/util/broadcasting.cpp [hunk 1]
```diff
@@ -153,16 +166,35 @@ static std::shared_ptr<ngraph::Node>
     }
 
     // Remove axes which have length of 1 from source shape
-    auto broadcasted_node = std::make_shared<ngraph::op::Reshape>(
-        node, ngraph::get_default_order(node->get_shape()), squeezed_shape);
+    ngraph::Output<ngraph::Node> broadcasted_value = std::make_shared<ngraph::op::Reshape>(
```
Comment (aprocter) [inline src/ngraph/op/util/broadcasting.cpp]: Out of scope for this PR, but why are we not `using namespace ngraph` in this file?

File: src/ngraph/op/util/broadcasting.cpp [hunk 2]
```diff
@@ -104,6 +104,19 @@ static std::pair<ngraph::Shape, std::vector<ngraph::Shape>>
     return get_numpy_broadcast_shapes(input_shapes);
 }
 
+static std::pair<ngraph::Shape, std::vector<ngraph::Shape>>
```
Comment (aprocter) [inline src/ngraph/op/util/broadcasting.cpp]: The duplication in this file is a bit of a drag, but I don't have any bright ideas. (Nor even any dim ones.)

File: src/ngraph/op/util/broadcasting.cpp (hunk 1 above)
Comment (diyessi) [reply src/ngraph/op/util/broadcasting.cpp]: I wondered the same. It's not alone.

File: src/ngraph/op/util/broadcasting.cpp (hunk 2 above)
Comment (diyessi) [reply src/ngraph/op/util/broadcasting.cpp]: The vector ones aren't as friendly.

Comment (aprocter) [reply src/ngraph/op/util/broadcasting.cpp]: FWIW, if `OutputVector` were its own class (is it not already?), we _could_ have an `as_single_output_nodes` akin to what we have for `Output`.

Comment (diyessi) [reply src/ngraph/op/util/broadcasting.cpp]: It's not.

Comment (aprocter) [reply src/ngraph/op/util/broadcasting.cpp]: I think this idea is probably a good one, but maybe in a later PR.

**Pull request merged**